tests/load/smoke.py

```diff
@@ -0,0 +1,54 @@
+from locust import HttpLocust, TaskSet, task
+
+
+class SmokeBehavior(TaskSet):
+    """
+    Request behavior that represents some standard production behavior.
+    Tasks uris are hard-coded for now, but could be stored or even dynamically
+    generated later.
+    See https://bugzil.la/1186085
+    """
+
+    @task(265)
```

Comment:
They are weights for task distribution during the test. These values are based on sampled traffic so this test sends requests proportionate to a snapshot of traffic levels on production. (See https://bugzilla.mozilla.org/show_bug.cgi?id=1186085#c1)